index.php

```diff
@@ -0,0 +1,60 @@
+<?php
+require_once __DIR__ . '/vendor/autoload.php';
+
+$db = new TopCat\Utilities\DB();
+$dbConnection = $db->dbConnect();
+$catHydrator = new TopCat\Hydrators\CatHydrator($dbConnection);
+
+// Populate the dropdown with the list of breeds from the DB
+$breeds = $catHydrator->getBreeds($dbConnection);
```

Comment:
Why are you passing in an unused argument? This method has no parameters

Reply:
doing